Modules/Tokens/src/Tokens.Application/Tokens/Commands/CreateToken/Validator.cs

```diff
@@ -18,5 +20,11 @@ public Validator()
 
         RuleFor(t => t.ExpiresAt)
             .GreaterThan(SystemTime.UtcNow).WithMessage("'{PropertyName}' must be in the future.").WithErrorCode(GenericApplicationErrors.Validation.InvalidPropertyValue().Code);
+
+        RuleFor(t => t.ForIdentity)
+            .NotEmpty()
+            .When(t => t.ForIdentity != null)
```

Comment:
Is this part actually necessary? Doesn't the ValidId check fail in case ForIdentity is empty?

Reply:
Yeah, it's not necessary and I was split on should I add it or not. It's advised as a good practice to not even run the more complex check if most basic one could fail. On the other hand, if there's a peace of code that could be removed and nothing in the project would change then that code should be removed. So, leave or remove?

Reply:
Removed.